ui/app/serializers/pki/action.js

```diff
@@ -54,6 +54,10 @@ export default class PkiActionSerializer extends ApplicationSerializer {
           'type',
           ...keyFields,
         ];
+      case 'generate-csr':
+        return ['common_name', ...keyFields];
```

Comment:
Heads up that this has been updated upstream so might be a good time to merge in main.

Reply:
Thank you! I had just merged main yesterday whenI added this. So many PRs!